src/Microsoft.Identity.Client/Cache/Items/MsalAccountCacheItem.cs

```diff
@@ -34,7 +34,6 @@
 
 namespace Microsoft.Identity.Client.Cache.Items
 {
-    [DataContract]
```

Comment:
>[DataContract] [](start = 4, length = 14)

isn't this needed for MSAL v2 cache? The dictionary serializer?

Reply:
I don't think so, none of the other cache items have this attribute.